tests/integrationv2/providers.py

```diff
@@ -146,6 +146,9 @@ def setup_client(self, options: ProviderOptions):
         if options.client_certificate_file:
             cmd_line.extend(['--cert', options.client_certificate_file])
 
+        if options.extra_flags is not None:
+            cmd_line.extend(options.extra_flags)
+
```

Comment:
So, the change was in this PR :)

Reply:
I haven't even merged this one yet :) This change was in like 3 PRs.